lib-database-entities/src/commonMain/kotlin/com/ustadmobile/lib/db/entities/School.kt

```diff
@@ -28,6 +28,12 @@ open class School() {
     //Active
     var schoolActive: Boolean = false
 
+    var schoolPhoneNumber : Long = 0L
```

Comment:
This should be a string to allow for + symbol, areas codes brackets etc. e.g. +1 (212) 123 4567